CatAnalyzer/plugins/h2muAnalyzer.cc

```diff
@@ -348,51 +405,19 @@ int h2muAnalyzer::JetCat_GC(float mu1_eta, float mu2_eta)
   float GC=0;
   for(int i=0; i<2; i++){
     if (eta_mu[i] < 0.8) GC += 1;
-    if (eta_mu[i] > 0.8 && eta_mu[i] < 1.6) GC += 10;
-    if (eta_mu[i] > 1.6 && eta_mu[i] < 2.4) GC += 100;
+    if (eta_mu[i] > 0.8 && eta_mu[i] < 1.5) GC += 10;
+    if (eta_mu[i] > 1.5 && eta_mu[i] < 2.4) GC += 100;
   }  
   return GC;
 }
 
-// ------------ method called once each job just before starting event loop  ------------
-void 
-h2muAnalyzer::beginJob()
-{
-}
-
-// ------------ method called once each job just after ending the event loop  ------------
-void 
-h2muAnalyzer::endJob() 
-{
-}
-
-// ------------ method called when starting to processes a run  ------------
-void 
-h2muAnalyzer::beginRun(edm::Run const&, edm::EventSetup const&)
-{
-}
-
-// ------------ method called when ending the processing of a run  ------------
-void 
-h2muAnalyzer::endRun(edm::Run const&, edm::EventSetup const&)
-{
-}
-
-// ------------ method called when starting to processes a luminosity block  ------------
-void 
-h2muAnalyzer::beginLuminosityBlock(edm::LuminosityBlock const&, edm::EventSetup const&)
-{
-}
-
-// ------------ method called when ending the processing of a luminosity block  ------------
-void 
-h2muAnalyzer::endLuminosityBlock(edm::LuminosityBlock const&, edm::EventSetup const&)
-{
-}
-
-// ------------ method fills 'descriptions' with the allowed parameters for the module  ------------
-void
-h2muAnalyzer::fillDescriptions(edm::ConfigurationDescriptions& descriptions) {
+void h2muAnalyzer::beginJob(){}
+void h2muAnalyzer::endJob(){}
```

Comment:
Better to remove unused functions :-)
beginJob and endJob are not automatically called by EDAnalyzer.